tests/test_orient_symmetric.py

```diff
@@ -1,28 +1,198 @@
 from unittest import TestCase
 
 import numpy as np
-from numpy import linalg
+from numpy import pi, random
+from numpy.linalg import norm
+from parameterized import parameterized
 
 from aspire.abinitio import CLSymmetryC3C4
 from aspire.source import Simulation
-from aspire.utils import J_conjugate, all_pairs
-from aspire.utils.random import randn
+from aspire.utils import Rotation
+from aspire.utils.misc import J_conjugate, all_pairs, gaussian_3d
+from aspire.volume import Volume
 
 
 class OrientSymmTestCase(TestCase):
+    # `order` is at this scope to be picked up by parameterized in testCommonLines.
+    order = 3
+
     def setUp(self):
-        self.L = 32
-        self.symm = "C4"
+        self.L = 64
         self.n_img = 32
-        self.seed = 8675309
-        src = Simulation(L=self.L, n=self.n_img, symmetry=self.symm)
-        self.cl_class = CLSymmetryC3C4(
-            src, symmetry=self.symm, n_theta=360, seed=self.seed
-        )
+        self.dtype = np.float32
+        self.n_theta = 360
+
+        # Build symmetric volume and associated Simulation object and common-lines class.
+        # For the Simulation object we use clean, non-shifted projection images.
+        orders = [3, 4]
+        self.vols = {}
+        self.srcs = {}
+        self.cl_classes = {}
+
+        for o in orders:
```

Comment:
It's a little dangerous to use `o` (or `O`) as a variable name. Maybe just write `order`?

Reply:
Thanks. Changed.